app/views/layouts/application.html.erb

```diff
@@ -16,27 +16,40 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
-<div class="container">
   <%# https://github.com/seyhunak/twitter-bootstrap-rails#using-helpers %>
-  <%= nav_bar do %>
-    <%= menu_group do %>
-      <%= menu_items %>
-      <% if user_signed_in? %>
-        <%= drop_down current_user.name do %>
-          <%= dropdown_items %>
-        <% end %>
-      <% else %>
-        <%= menu_item t(:login, scope: 'navbar'), new_user_session_path %>
+<%= nav_bar brand: t(:app_name), responsive: true, static: :top do %>
```

Comment:
The `responsive` attribute makes no sense as long as you don't use responsive features at all.